README.md

```diff
@@ -1 +1,93 @@
-![](https://travis-ci.org/larsbrinkhoff/lbForth.svg?branch=master)
+# `supdup`
+
+## Overview
+
+* `supdup` is a client (and `supdupd` is a server) implementing the SUPDUP (*Sup*er *Dup*er TELNET) protocol, a highly efficient [TELNET](https://www.rfc-editor.org/rfc/rfc854.txt)-style remote display protocol, on UNIX-like systems.
+
+* It originated as a private protocol between the [Chaosnet](https://chaosnet.net/)-connected ITS systems at MIT to allow a user at any one of these systems to use one of the others as a display; later implementations were developed for [various platforms](https://gunkies.org/wiki/Chaosnet#Protocol_implementations).
+
+* The software supports connections over TCP/IP and Chaosnet (optionally using the [cbridge](https://github.com/bictorv/chaosnet-bridge) software).
+
+## Supported platforms
```

Comment:
We don't _support_ anything, really.  Please rephrase.

Reply:
Done.